web/src/lib/components/shared-components/notification/notification-list.svelte

```diff
@@ -1,25 +1,21 @@
 <script lang="ts">
-	import { ImmichNotification, notificationController } from './notification';
+	import { notificationController } from './notification';
 	import { fade } from 'svelte/transition';
 
 	import NotificationCard from './notification-card.svelte';
 	import { flip } from 'svelte/animate';
 	import { quintOut } from 'svelte/easing';
 
-	let notificationList: ImmichNotification[] = [];
-
-	notificationController.notificationList.subscribe((list) => {
-		notificationList = list;
-	});
+	const { notificationList } = notificationController;
```

Comment:
I love this so much since I discovered I could just destruct a variable to use the Writable property directly without subscribing 😝. Thanks Jaime

Reply:
Yes very handy, and Svelte takes care of unsubscribing automatically 😄